### DISCUSSION TITLE
Adafruit 7-segment LED backpack driver worth including?

### DISCUSSION THREAD

**Discussion by chudsaviet** [UncleRus/esp-idf-lib#342] [Ideas]
Hi!  
I have recently prepared an HT16K33 pull request - https://github.com/UncleRus/esp-idf-lib/pull/341  
Actually, it a foundation for a driver for Adafruit 7-segment LED backpacks, like https://www.adafruit.com/product/1269 .  
I will make the driver, but the question is:  
Is it general enough to be included in this library? These backpacks are from one manufacturer operating mainly in US.

**Comment by UncleRus** [comment]
As for me, the basic driver is enough. If we start to take into account all the options for the layout of the LEDs, then this will be fundamentally wrong. Let the author of the firmware for a specific hardware do this, and not the author of the driver.

**Reply by chudsaviet** [reply]
Yep, thats mostly my thoughts.
